React assignment 3

abhishekpatel946 opened the pull request:
### Let’s create some form components.
- [x] Component for Input text
- [x] Component for Input file
- [x] Component for Getting Date
- [x] Component For Select Options
- [x] Component for showing options in the Radio button
- [x] Components for range

Pending Task
- [ ] SCSS not implemented.

abhishekpatel946 committed:
Initialize project using Create React App

abhishekpatel946 committed:
feat: react form with plain CSS

abhishekpatel946 committed:
fix: resolve unrelated histories conflict from ReadME.md

abhishekpatel946 committed:
feat: deployed using Netlity & update link in ReadME

abhishekpatel946 committed:
fix: update CSS prop for formLayout

abhishekpatel946 committed:
fix: demo.png link is working in ReadME.md

the link was broken before & demo.png was not shown up in ReadME for demo the App.

hrathi22 commented on src/Form/css/Form-style.css:
```diff
@@ -0,0 +1,97 @@
+  color: lightcyan;
+}
+select option[value='op1'] {
+  background-color: transparent;
```
Why diff style for same options?

hrathi22 commented on src/Utils/DateUtil.jsx:
```diff
@@ -0,0 +1,17 @@
+
+const DateUtil = () => {
+  // date method for current localDate (dd-mm-yyyy)
+  const today = new Date().toLocaleDateString();
```
Each time the component will re-render the date will gate change, manage it accordingly

abhishekpatel946 replied:
Sir, I didn't get your question. 
DateUtil just a component to render the current date whenever we use it.

hrathi22 commented on src/Form/Label.jsx:
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './css/Form-style.css';
+// import '../css/main.css';
```
remove unused imports

hrathi22 commented on src/Container/layout-style.css:
```diff
@@ -0,0 +1,12 @@
+}
+
+.Layout-container {
+  /* height: 100px; */
```
Remove commented code

hrathi22 commented on src/Form/InputFile.jsx:
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Create new folder named components and place Form there

hrathi22 commented on src/Form/Label.jsx:
```diff
@@ -0,0 +1,13 @@
+import './css/Form-style.css';
+// import '../css/main.css';
+
+const Label = (props) => {
```
add proptypes

abhishekpatel946 committed:
fix: Update after review

remove repeated css code, remove the commented code, create a components dir to contain the forms

abhishekpatel946 committed:
fix: handle DateUtil change

useState & useEffect will detect the changes in date and change the state of date immediately, no need to re-rendering

abhishekpatel946 committed:
fix: handle PropTypes checking

strictly checking the label name in each component using proptype

abhishekpatel946 committed:
Merge branch 'react_assignment_3' of https://github.com/abhishekpatel946/React-Assignment-3 into react_assignment_3
